bigquery_etl/schema/stable_table_schema.py

```diff
@@ -0,0 +1,131 @@
+"""Methods for working with stable table schemas."""
```

Comment:
Moved out from `bigquery_etl/view/generate_stable_views.py`. These methods are used in a couple of places.